src/test/java/net/minecraftforge/debug/entity/player/SwimSpeedAttributeTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016.
```

Comment:
Please update this to follow the new copyright format that has recently rolled out.